lib/views/auth/login_view.dart

```diff
@@ -76,37 +75,38 @@ class LoginScreenState extends State<LoginScreen> {
                           height: height * 0.020,
                         ),
                         Stack(
-                            // ignore: always_specify_types
-                            children: [
-                              Transform.translate(
-                                offset: const Offset(25, 10),
-                                child: Container(
-                                  color: Colors.black,
-                                  width: width * 0.77,
-                                  height: height * 0.065,
-                                ),
+                          // ignore: always_specify_types
+                          children: [
+                            Transform.translate(
+                              offset: const Offset(25, 10),
+                              child: Container(
+                                color: Colors.black,
+                                width: width * 0.77,
+                                height: height * 0.065,
                               ),
-                              Padding(
-                                padding: const EdgeInsets.only(
-                                    left: 20.0, right: 20.0),
-                                // ignore: sized_box_for_whitespace
-                                child: Container(
-                                  height: height * 0.07,
-                                  child: TextField(
-                                      controller: _emailController,
-                                      decoration: const InputDecoration(
-                                          labelText: 'Email Address',
-                                          labelStyle: TextStyle(
-                                              fontSize: 20.0,
-                                              color: Colors.black),
-                                          filled: true,
-                                          fillColor: Colors.white,
-                                          border: OutlineInputBorder(
-                                              borderRadius:
-                                                  BorderRadius.zero))),
+                            ),
```

Comment:
What changes were required here? I don't see any fixes. If you want to formate the code use CTRL + F